libraries/riscv-csr/src/csr.rs

```diff
@@ -20,8 +20,8 @@ pub const MEPC: usize = 0x341;
 pub const MCAUSE: usize = 0x342;
 pub const MTVAL: usize = 0x343;
 pub const MIP: usize = 0x344;
-pub const MSECCFG: usize = 0x390;
-pub const MSECCFGH: usize = 0x391;
+pub const MSECCFG: usize = 0x747;
+pub const MSECCFGH: usize = 0x757;
```

Comment:
This is always concerning. It would be nice to at some point split out the riscv stable CSRs from the experimental, chip-specific ones so this patch wouldn't look so problematic.